soroban-sdk/src/testutils.rs

```diff
@@ -191,3 +157,18 @@ pub(crate) fn random<const N: usize>() -> [u8; N] {
     rand::thread_rng().fill_bytes(&mut arr);
     arr
 }
+
+#[cfg(not(target_family = "wasm"))]
```

Comment:
Why do we need the wasm cfg?

Reply:
```suggestion
```

Reply:
I added this because `ScVal` conversions are conditioned on `not(wasm)`, while both of the functions utilize these. So IIUC the build won't compile in test/testutils mode with wasm target. That's probably not a big deal, given that we don't normally build this configuration, but I think it would be nice to have all the feature combinations and archs to compile. Maybe `ScVal` conversions should be gated by the test/testutils and not target?

Reply:
Yeah it seems like overtime we've ended up with two ways to specify the same thing. This came about because originally the sdk was going you be used for native contracts too. In native contracts ScVal conversions would be supported, but not testutils.

Note that it's not possible to enable testutils on Wasm builds due you the condition below. So while none of this is perfect, functionally there's no impact.

https://github.com/stellar/rs-soroban-sdk/blob/e7910c7057088b3418d84af04f2c04313d0c756f/soroban-sdk/src/lib.rs#L45-L46

Reply:
Cool, then this is good enough for now.